### PR TITLE
Add Uno icon

### PR DESCRIPTION
**Pull request by clairekardas**
uno

**Issue:**
**Alexa rank: 332,428 (https://www.letsplayuno.com/)**
  

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description

- Extracted logo SVG from Wikipedia
- Simplified and flattend the original 3D-optic logo manually
- Color sourced from the circular background of the colored icon

Note: https://www.letsplayuno.com/ uses an exclamation mark in their logo (Uno!), however it seems that this version is merely used for their mobile game app icon. hasbro.com and wikipedia.org both use the more known version, without an exclamation mark. It hasn’t been changed anywhere else, notably not on the for-sale paper version of the game. Also, there is no SVG version of the version with exclamation mark available and it doesn’t show up in google image search results. This is why I did not use the exclamation mark version.

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -9632,6 +9632,11 @@
             "hex": "000000",
             "source": "https://unity.com/"
         },
+        {
+            "title": "Uno",
+            "hex": "ED1C24",
+            "source": "https://en.wikipedia.org/wiki/File:UNO_Logo.svg"
```
Comment (adamrusted) [inline _data/simple-icons.json:9638]:
```suggestion
            "source": "https://commons.wikimedia.org/wiki/File:UNO_Logo.svg"
```

File: icons/uno.svg
Comment (adamrusted) [inline icons/uno.svg:1]: I would probably expect to see the oval behind the text, as it appears most places. See below for a rough idea.

uno

SVG code below, if you'd like to use it.
```html
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Uno</title><path d="M15.217 3.332c-1.928.004-4.007.481-5.982 1.53-2.618 1.3-4.722 3.205-6.053 5.253a.36.36 0 0 1 .213.217l1.69 4.521c.096.261.234.509.426.674a.864.864 0 0 0 .536.22h.137c0-.083-.013-.165-.027-.248-.028-.124-.055-.247-.11-.385l-1.69-4.521a.36.36 0 0 1 .11-.413l.92-.92a.375.375 0 0 1 .165-.124l1.996-.743a.273.273 0 0 1 .12-.068l1.829-.673a.353.353 0 0 1 .33.041l1.786 1.168-.33-.88a.36.36 0 0 1 .083-.384l.962-.976a.562.562 0 0 1 .123-.069l2.09-.783a.362.362 0 0 1 .467.22c0 .014.013.014.013.027l.225.6c.155-.213.332-.43.531-.64.399-.427.88-.84 1.416-1.197.137-.096.288-.178.44-.26.206-.083.37-.166.535-.22a4.65 4.65 0 0 1 .986-.25 10.784 10.784 0 0 0-3.937-.717Zm4.576 1.046a4.114 4.114 0 0 0-1.487.264c-.316.11-.619.261-.88.44-1.038.683-1.849 1.617-2.254 2.383l-.508-1.352-2.089.783-.962.976.715 1.91L9.62 8.009l-1.841.673-2.09.784-.975.976 1.69 4.521c.11.275.165.564.151.852-.054.248-.316.275-.508.261-.673-.027-1.072-.536-1.292-1.113l-1.69-4.521-2.103.783-.962.976 1.65 4.425c.645 1.745 1.813 2.804 3.462 2.707 1.072-.055 2.213-.536 2.969-1.333.48-.48 1.052-.987 1.31-1.419l.408 1.09 2.102-.798.976-.962h-.013l-.77-2.089 2.872 1.91 1.732-.631.976-.976-.12-.316c.893.348 1.915.385 2.95 0 .949-.358 1.925-1.004 2.873-2.57.041-.069.083-.138.11-.206.59-1.086.687-2.46.206-3.739-.644-1.728-2.17-2.895-3.9-2.916zm-.099.278c1.657-.022 3.127 1.084 3.752 2.748.77 2.076-.028 4.384-2.172 5.195-2.144.797-4.26-.426-5.043-2.501-.77-2.089.013-4.398 2.171-5.195a3.85 3.85 0 0 1 1.292-.247zm-.021 1.607a2.141 2.141 0 0 0-.708.138c-1.305.481-1.58 1.828-1.126 3.037.453 1.21 1.525 2.034 2.844 1.553 1.306-.494 1.58-1.828 1.127-3.023-.38-.983-1.158-1.72-2.137-1.705Zm-5.174.207 2.817 7.49-1.347.495-5.236-3.477 1.787 4.768-1.594.591-2.79-7.49 1.443-.536 5.057 3.313-1.704-4.563Zm4.263 1.05c.136 0 .283.022.437.063.632.165 1.113.756 1.374 1.443.055.137.097.275.124.426.124.564.069.962-.165 1.182-.275.261-.783.275-1.36.027-.481-.26-.852-.742-1.072-1.319a2.613 2.613 0 0 1-.179-1.237c0-.013.014-.027.014-.04.113-.372.42-.55.827-.546zm.03.353a.743.743 0 0 0-.266.04.385.385 0 0 0-.22.22l-.028.041c-.013.165-.013.33.014.495a2.549 2.549 0 0 0 .522 1.237c.138.178.303.316.495.426.234.096.454.15.632.137a.514.514 0 0 0 .344-.11.415.415 0 0 0 .11-.247c.027-.151 0-.344-.041-.591l-.042-.193c-.013-.055-.04-.123-.055-.178a2.585 2.585 0 0 0-.494-.825 1.414 1.414 0 0 0-.633-.398 1.36 1.36 0 0 0-.338-.054ZM7.614 9.053l1.58 4.22c.742 1.979.344 3.587-1.677 4.452l-.178.07c-.014 0-.028.013-.041.013-.014 0-.028.014-.042.014l-.178.068c-2.09.66-3.477-.343-4.206-2.322L1.32 11.404l1.58-.591 1.595 4.26c.261.687.756 1.264 1.54 1.292a1.68 1.68 0 0 0 .686-.11c.22-.083.44-.206.605-.371.564-.536.564-1.292.302-1.98l-1.594-4.26Zm12.99 5.098-.003.001a4.48 4.48 0 0 1-2.634.185.302.302 0 0 1-.046.049l-.976.976a.285.285 0 0 1-.137.082l-1.718.632a.353.353 0 0 1-.33-.041l-1.938-1.292.371.976a.447.447 0 0 1 .055.192.322.322 0 0 1-.11.248l-.975.976a.325.325 0 0 1-.124.082L9.936 18a.362.362 0 0 1-.467-.22l-.175-.466c-.204.227-.44.457-.677.686-.096.097-.206.193-.289.275l-.013.014a3.158 3.158 0 0 1-.564.481c-.7.495-1.54.81-2.419.907a3.95 3.95 0 0 1-.938-.013c2.585 1.492 6.457 1.356 10.504-.647 2.344-1.16 4.315-2.959 5.707-4.866z"/></svg>
```

**Review by adamrusted — CHANGES_REQUESTED** [review]

File: icons/uno.svg
Comment (clairekardas) [reply icons/uno.svg:1]: LGTM!

Comment (adamrusted) [reply icons/uno.svg:1]: Awesome! Do you want me to merge this new version in? Happy to, and then we'll need another maintainer to review :)

File: _data/simple-icons.json [hunk 2]
```diff
@@ -12551,11 +12551,17 @@
...
+            "title": "Uno",
+            "hex": "ED1C24",
+            "source": "https://commons.wikimedia.org/wiki/File:UNO_Logo.svg"
+        },
         {
             "title": "UnoCSS",
             "hex": "333333",
             "source": "https://github.com/unocss/unocss/blob/fc2ed5bb6019b45565ff5293d4b650522f1b79b4/playground/public/icon.svg"
         },
+
```
Comment (adamrusted) [inline _data/simple-icons.json:12564]:
```suggestion
```

**Review by adamrusted — CHANGES_REQUESTED** [review]
Linter is failing because of the below. If you can fix this, the PR should be good for another @simple-icons/icon-review member to review it.

**Comment by clairekardas** [comment]
> Linter is failing because of the below. If you can fix this, the PR should be good for another @simple-icons/icon-review member to review it.

Thank you, I don't know how I didn't spot that 😅

**Comment by LitoMore** [comment]
Thank you for submitting this. I love Uno®!

But I don't know if this icon can be added to our collection. Mattel owns the Uno® trademark, and here is their [Term and Conditions](https://shop.mattel.com/pages/terms-conditions)

Another point is their trademark in multiple colors. But our project only stores monochrome shapes. It's better to find out a use case they use the monochrome shape on their products like cards or games etc.

I own this game on Nintendo Switch™ and PlayStation®, I can take a look on these platforms.

@simple-icons/maintainers thoughts?

**Comment by adamrusted** [comment]
Given no feedback, how should we proceed on this one @LitoMore?

**Review by LitoMore — APPROVED** [review]
It looks good to me. This game on PlayStation is using the same icon.

**Comment by clairekardas** [comment]
Thanks for all the work here, guys!

**Pull request merged**